external_deps/build.sh

```diff
@@ -903,40 +921,78 @@ setup_linux-arm64-default() {
 	common_setup linux aarch64-unknown-linux-gnu
 }
 
+base_windows_amd64_msvc_packages='pkgconfig zlib gmp nettle curl sdl2 glew png jpeg webp freetype openal ogg vorbis opus opusfile lua naclsdk naclports genlib'
+all_windows_amd64_msvc_packages="${base_windows_amd64_msvc_packages}"
+
+base_windows_i686_msvc_packages="${base_windows_amd64_msvc_packages}"
+all_windows_i686_msvc_packages="${base_windows_amd64_msvc_packages}"
+
+base_windows_amd64_mingw_packages='zlib gmp nettle curl sdl2 glew png jpeg webp freetype openal ogg vorbis opus opusfile lua naclsdk naclports'
+all_mingw_packages="${base_windows_amd64_mingw_packages}"
+
+base_windows_i686_mingw_packages="${base_windows_amd64_mingw_packages}"
+all_windows_i686_mingw_packages="${base_windows_amd64_mingw_packages}"
+
+base_macos_amd64_default_packages='pkgconfig nasm gmp nettle sdl2 glew png jpeg webp freetype openal ogg vorbis opus opusfile lua naclsdk naclports'
+all_macos_amd64_default_packages="${base_macos_amd64_default_packages}"
+
+base_linux_amd64_default_packages='naclsdk naclports'
+all_linux_amd64_default_packages='zlib gmp nettle curl sdl2 glew png jpeg webp freetype openal ogg vorbis opus opusfile lua naclsdk naclports'
+
+base_linux_i686_default_packages="${base_linux_amd64_default_packages}"
+all_linux_i686_default_packages="${all_linux_amd64_default_packages}"
+
+base_linux_arm64_default_packages='naclsdk'
+all_linux_arm64_default_packages='zlib gmp nettle curl sdl2 glew png jpeg webp freetype openal ogg vorbis opus opusfile lua naclsdk'
+
+base_linux_armhf_default_packages="${base_linux_arm64_default_packages}"
+all_linux_armhf_default_packages="${all_linux_arm64_default_packages}"
+
 # Usage
 if [ "${#}" -lt "2" ]; then
-	cat <<-EOF
-	usage: ${0} <platform> <package[s]...>
+	sed -e 's/\\t/  /g' <<-EOF
```

Comment:
It allows me to use `\t` when I want a tab, whatever the final tab format used, instead of hardcoding two spaces or something like that, and it makes sure all the tabs are the sames when printed.

Reply:
In the end I translated to a real tab. This is what I always do in scripts.

Reply:
In that case it is using the same kind of indentation as the source file so there is no reason to not indent it in the source.

Reply:
The `-` in ` <<-EOF` means all the leading tabs from the line are stripped and are not part of the heredoc, that's the mechanism allowing to ident the heredoc in a if block (or in a function, etc.).

Reply:
Well now I'm thinking if you have to use sed you should use it to strip a leading tab instead of the minus thingy... :laughing: but LGTM

Reply:
LOL, that would work indeed. 🤯️ But would not survive a indentation change, so hacky! 😁️